src/css/controls/imports/settings-menu.less

```diff
@@ -180,7 +180,7 @@
         }
     }
 
-    .jw-breakpoint-7 &:not(.jw-tizen-app &) {
```

Comment:
Instead of complicating these selectors (making them more specific which could override other selectors), how about having the tizen controls not apply these breakpoint classes? Just apply the breakpoint(s) class you want in tizen controls by overriding the method that is adding them in controls. This is something that in general we should try to do to simplify our selectors.

Reply:
I felt the same way which is why I am removing all the `not(.jw-tizen-app)` selectors from the css classes. These were initially added for quick styling in the first app. All the tizen styling is in `tizen.less` now and the fullscreen flag is added in tizen-controls.

Reply:
Sorry I was commenting on something you fixed, as if it was a new change. ☕☕☕ Nice fix.